neptune_notebooks/sync.py

```diff
@@ -0,0 +1,162 @@
+#
+# Copyright (c) 2019, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import json
+import os
+import sys
+
+import click
+
+import neptune
+from neptune.api_exceptions import NotebookNotFound
+
+
+def sync(project_name, path, update_flag=None, new_flag=None):
+    verify_file(path)
+    absolute_path = os.path.abspath(path)
+    project = neptune.init(project_qualified_name=project_name)
+    notebook_json = load_notebook_json(path)
+    nbformat = int(notebook_json["nbformat"]) if "nbformat" in notebook_json else 0
```

Comment:
This will throw an exception when converting to int for invalid jsons, we should handle it gracefully